src/apps/common/submodules/mediaSelect/mediaSelect.js

```diff
@@ -294,6 +313,46 @@ define(function(require) {
 			monster.ui.fancyTabs(template.find('.monster-tab-wrapper'));
 
 			self.mediaSelectBindCommon(template, mediaToUpload);
+		},
+
+		mediaSelectUpdateTTSMedia: function(args, callback) {
+			var self = this,
+				greetingMedia = {
+					description: '<Text to Speech>',
```

Comment:
If this is displayed to the user, it should be part of the `i18n` files.

Reply:
No, it isn't

Reply:
Yeah, that [prop](https://docs.2600hz.com/supported/applications/crossbar/doc/media/#schema) is usually used to store the original file name, which doesn't really apply to TTS so we use a placeholder instead.